zubbi/scraper/scraper.py

```diff
@@ -178,7 +206,8 @@ def scrape_role_files(self):
         except CheckoutError as e:
             LOGGER.debug(e)
 
-        return role_files
+        # sort keys (role names) alphabetically
```

Comment:
this was old behavior when you listed only dirs in ROLES directory, so this preserves the functionality, plus it is useful for tests, because we know the eventual order of retrieved roles. Without sorting it would depend on the number of nested directories